CRM/Civicase/Settings.php

```diff
@@ -30,30 +41,34 @@ public static function getAll(): array {
       'caseCategoryInstanceType' => 'case_category_instance_type',
     ];
 
+    OptionValuesHelper::setToJsVariables($options);
+
     [$caseCategoryId, $caseCategoryName] = CaseUrlHelper::getCategoryParamsFromUrl();
 
-    CRM_Civicase_Hook_Helper_CaseTypeCategory::addWordReplacements($caseCategoryName);
-    $permissionService = new CaseCategoryPermission();
-    $caseCategoryPermissions = $permissionService->get($caseCategoryName);
     $caseCategorySetting = new CRM_Civicase_Service_CaseCategorySetting();
-
-    OptionValuesHelper::setToJsVariables($options);
     NewCaseWebform::addWebformDataToOptions($options, $caseCategorySetting);
-
-    CaseCategoryHelper::updateBreadcrumbs($caseCategoryId);
+    $permissionService = new CaseCategoryPermission();
+    $caseCategoryPermissions = $permissionService->get($caseCategoryName);
     self::setCaseActions($options, $caseCategoryPermissions);
     self::setContactTasks($options);
     self::setCaseTypesToJsVars($options);
     self::setCaseCategoryInstanceToJsVars($options);
     self::setRelationshipTypesToJsVars($options);
     self::setFileCategoriesToJsVars($options);
     self::setActivityStatusTypesToJsVars($options);
-    self::setCustomFieldsInfoToJsVars($options);
     self::setTagsToJsVars($options);
     self::addCaseTypeCategoriesToOptions($options);
     self::exposeSettings($options, ['caseCategoryId' => $caseCategoryId]);
 
-    return $options;
+    if (str_contains(CRM_Utils_System::currentPath(), '/case')) {
```

Comment:
@shahrukh-compuco  does this mean `addWordReplacement` function will not be called on the contact view page?

Reply:
@shahrukh-compuco  have you tried to create a new case category?